repoclosure/yum.conf

```diff
@@ -148,18 +148,18 @@ baseurl=http://koji.katello.org/releases/yum/katello-nightly/candlepin/el8/x86_6
 [el6-base]
 name=BaseOS
 enabled=1
-mirrorlist=http://mirrorlist.centos.org/?release=6&arch=x86_64&repo=os
+baseurl=https://vault.centos.org/6.10/os/x86_64/
 failovermethod=priority
 
 [el6-updates]
 name=updates
 enabled=1
-mirrorlist=http://mirrorlist.centos.org/?release=6&arch=x86_64&repo=updates
+baseurl=https://vault.centos.org/6.10/updates/x86_64/
 failovermethod=priority
 
 [el6-extras]
 name=extras
-mirrorlist=http://mirrorlist.centos.org/?release=6&arch=x86_64&repo=extras
+baseurl=https://vault.centos.org/6.10/extras/x86_64/
 
 [el6-epel]
```

Comment:
This is also empty now and should be removed or at least replaced with something that does work.

Reply:
replaced (can't be removed as the openscap client depends on rubygem-json that's in epel)